docs/grid_exchange_formats/cgmes/export.md

```diff
@@ -1,8 +1,154 @@
+In both cases, the metadata model information in the exported files is built from metadata information read from the input files and stored in IIDM or received through parameters. 
+Information received through parameters takes precedence over information available from original metadata models.
+
+For a quick CGM export, the user may rely on the parameter **iidm.export.cgmes.cgm_export** to write in a single export multiple updated SSH files (one for each IGM) and a single SV for the whole common grid model. Specifics about this option are explained in the section [below](#cgm-common-grid-model-quick-export).
+If you need complete control over the exported files in a CGM scenario, you may prefer to iterate through the subnetworks and make multiple calls to the export function. This is described in detail in the section [below](#cgm-common-grid-model-manual-export).    
+
+Please note that when exporting equipment, PowSyBl always use the CGMES node/breaker level of detail, without considering the topology
+level of the PowSyBl network.
+
+The user can specify the profiles to be exported using the parameter **iidm.export.cgmes.profiles**. The list of currently supported export instance files are: EQ, SSH, SV, TP. 
+
+If the IIDM network has at least one voltage level with node/breaker topology level, and the SSH or SV is requested in the export, and the TP is not requested, an error will be logged, as there could be missing references in the SSH, SV files to Topological Nodes calculated automatically by IIDM that are not present in the output.
+
+If the dependencies have to be updated automatically (see parameter **iidm.export.cgmes.update-dependencies** below), the exported instance files will contain metadata models where:
+* TP and SSH depend on EQ.
+* SV depends on TP and SSH.
+* EQ depends on EQ_BD (if present). EQ_BD is the profile for the boundary equipment definitions.
+* SV depends on TP_BD (if present). TP_BD is the profile for the boundary topology. Only for CGMES 2.4.
+
+The output filenames will follow the pattern `<baseName>_<profile>.xml`. The basename is determined from the parameters, or the basename of the export data source or the main network name. 
+
+## CGM (Common Grid Model) quick export
+
+When exporting a CGM, we need an IIDM network (CGM) that contains multiple subnetworks (one for each IGM). 
+Only the CGMES instance files corresponding to SSH and SV profiles are exported:
+an updated SSH file for every subnetwork (for every IGM) and a single SV file for the main network that represents the CGM.
+
+When exporting, it is verified that the main network and all subnetworks have the same scenario time (network case date). If they are different, an error is logged.
+
+If no version is provided as a parameter for the exported files, the output version is determined based on the max version of input CGM SV metadata and IGM SSH versions.
```

Comment:
the max version of input CGM SV metadata and IGM SSH versions -> not very clear.